cli/tools/init/mod.rs

```diff
@@ -238,6 +247,57 @@ Deno.test(function addTest() {
   Ok(())
 }
 
+pub async fn init_npm(name: &str, args: Vec<String>) -> Result<i32, AnyError> {
+  let script_name = format!("npm:create-{}", name);
+
+  fn print_manual_usage(script_name: &str, args: &[String]) -> i32 {
+    log::info!("{}", cformat!("You can initialize project manually by running <u>deno run {} {}</> and applying desired permissions.", script_name, args.join(" ")));
+    1
+  }
+
+  if std::io::stdin().is_terminal() {
+    log::info!(
+      cstr!("Deno requires <u>all permissions</> to create a new <g>{}</> project. Do you want to continue? <p(245)>[y/n]</>"),
```

Comment:
Maybe this could be more scary:

⚠️ Warning: Do you fully trust the `npm:create-vite` package? Deno will invoke code from it with all permissions. Use `--allow-read` to bypass this. 
do you want to continue? y/n

Reply:
`--allow-read` flag doesn't work in `deno init` and we have no way to know upfront which flags to pass.

I can certainly make it more scary